src/gromacs/nblib/CMakeLists.txt

```diff
@@ -38,13 +38,47 @@
 # \author Sebastian Keller <keller@cscs.ch>
 #
 
-file(GLOB NBLIB_SOURCES *.cpp)
+add_library(nblib "")
 
-install(FILES
```

Comment:
This is a much better approach

Reply:
Sorry, are you saying `install(...)` is a better approach than `target_sources(...)`? Could you elaborate or link some posts that explain your reasoning?

Reply:
At any rate, I mainly removed this in this PR since it is not used yet anyway.

Reply:
I think there is confusion here arising from the fact that I removed the `install(...)` lines because they are not currently used or needed. They would eventually be added back later when the complete library is exposed. There is no attempt to replace `install(...)` with `target_sources(...)` as part of this change. I was just not paying attention to the comment because I had a meeting to go to. I am marking this as resolved since there is not really any discussion warranted about removing this line of dead code.